### DISCUSSION TITLE
Sleep/Standby mode?

### DISCUSSION THREAD

**Discussion by lgarrido** [AxisCommunications/discussions#6] [VAPIX] [answered]
Hi!

We are trying to use AXIS cameras (model V5925) to record meetings.

In order to save energy, is it possible to use VAPIX to make a camera enter some sort of sleep mode whenever there are no meetings going on, and wake it up when it's needed? We work with different brands of cameras and all of them so far have offered this possibility.

VAPIX's Power Settings API doesn't seem to provide any such calls, and web search engines haven't yielded any result either for us.

Thanks in advance,

L

**Comment by mattias-kindborg-at-work** [comment]
Hi @lgarrido! We'll look into this and hopefully return with an answer next week.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @lgarrido.

It seems that there is no way to set the camera in sleep or hibernation mode, sorry.

**Comment by lgarrido** [comment]
Hi @mattias-kindborg-at-work, thanks for the answer.

Is there a reason behind the lack of that feature? Security concerns, perhaps? Or is it something that might be implemented in the future? It's a really useful feature for the kind of solutions we develop for our customers.

**Comment by mattias-kindborg-at-work** [comment]
Hi @lgarrido. A majority of the network cameras we sell are being used in the domain of safety and security. I would assume that this feature hasn't been asked for by these customers.
